src/main/java/com/loofah/graph/api/queries/SkillsQuery.java

```diff
@@ -13,6 +17,7 @@
 public class SkillsQuery implements DataFetcher<List<Skill>> {
 
     private final SkillRepository skillRepository;
+    private final ObjectMapper objectMapper = new ObjectMapper();
```

Comment:
Nitpick: if we made this a bean and injected it into the constructor, we could mock it where necessary in unit tests. e.g. to simulate any errors while converting

Reply:
yep makes sense, it's meant i have to add some more when's into the SkillsQueryTest and not sure it's completely necessary - sure there will be a chance to adjust that on a future ticket